python/paddle/fluid/contrib/slim/tests/QAT_mkldnn_int8_readme.md

```diff
@@ -1,58 +1,105 @@
+
+### Applying quantization passes
+
+Having gathered all the data needed for quantization we apply the `cpu_quantize_pass` which quantizes the graph, and the `cpu_quantize_squash_pass` which optimizes the INT8 graph.
+
+## 4. Code example
+
+The code snipped shows how the `Qat2Int8MkldnnPass` can be applied to a model graph:
 
 ```python
     import paddle.fluid as fluid
-    from paddle.fluid.contrib.slim.quantization import QatInt8MkldnnPass
     from paddle.fluid.contrib.slim.quantization import Qat2Int8MkldnnPass
     from paddle.fluid.framework import IrGraph
     from paddle.fluid import core	
     
     # Create the IrGraph by Program
     graph = IrGraph(core.Graph(fluid.Program().desc), for_test=False)
     place = fluid.CPUPlace()
-    # Convert the IrGraph to MKL-DNN supported INT8 IrGraph by using
-    # QAT1.0 MKL-DNN
-    # QatInt8MkldnnPass
-    mkldnn_pass = QatInt8MkldnnPass(fluid.global_scope(), place)
-    # Apply QatInt8MkldnnPass to IrGraph
-    mkldnn_pass.apply(graph)
-    # QAT2.0 MKL-DNN
-    # Qat2Int8MkldnnPass, it requires a list of operators to be quantized
+    # Convert the IrGraph to MKL-DNN supported INT8 IrGraph using the
+    # Qat2Int8MkldnnPass. It requires a list of operators to be quantized
     mkldnn_pass = Qat2Int8MkldnnPass({'conv2d', 'pool2d'}, fluid.global_scope(), place, fluid.core, False)
     # Apply Qat2Int8MkldnnPass to IrGraph
     mkldnn_pass.apply(graph)
 
 ```
 
-## 2. Accuracy and Performance benchmark
-
->**I. QAT1.0 MKL-DNN Accuracy on Intel(R) Xeon(R) Gold 6271**
-
-|     Model    | Fake QAT Top1 Accuracy | INT8 QAT Top1 Accuracy | Top1 Diff | Fake QAT Top5 Accuracy | INT8 QAT Top5 Accuracy | Top5 Diff |
-|:------------:|:----------------------:|:----------------------:|:---------:|:----------------------:|:----------------------:|:---------:|
-|   GoogleNet  |         70.40%         |         70.39%         |   -0.01%  |         89.46%         |         89.46%         |   0.00%   |
-| MobileNet-V1 |         70.84%         |         70.85%         |   +0.01%  |         89.59%         |         89.58%         |   -0.01%  |
-| MobileNet-V2 |         72.07%         |         72.06%         |   -0.01%  |         90.71%         |         90.69%         |   -0.02%  |
-|  ResNet-101  |         77.49%         |         77.52%         |   +0.03%  |         93.68%         |         93.67%         |   -0.01%  |
-|   ResNet-50  |         76.61%         |         76.62%         |   +0.01%  |         93.08%         |         93.10%         |   +0.02%  |
-|     VGG16    |         72.71%         |         72.69%         |   -0.02%  |         91.11%         |         91.09%         |   -0.02%  |
-|     VGG19    |         73.37%         |         73.37%         |   0.00%   |         91.40%         |         91.41%         |   +0.01%  |
+## 5. Accuracy and Performance benchmark
```

Comment:
Table I, II and III present the accuracy achieved by QAT2 method in the various configuration: (....)